types/joint.d.ts

```diff
@@ -454,299 +452,310 @@ export namespace dia {
         options?: object;
     }
 
-    class Paper extends Backbone.View<Graph> {
-        constructor(options?: PaperOptions);
+    var Paper: PaperStatic;
 
-        options: PaperOptions;
-        svg: SVGElement;
-        viewport: SVGGElement;
-        defs: SVGDefsElement;
+    interface PaperStatic {
```

Comment:
If we revert this back to a class, then we don't need this, and should put the constructor back.